rosplan_action_interface/src/RPKnowledgeBaseLink.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
```

Comment:
Is the rosplan_action_interface package the best one where this KBlink file should be saved?

Reply:
A dedicated KB python interface would be good to add - basically what DGerod has made. It would go in the KB package and imported, I would guess. That's a more significant endeavour we should add as a future PR.